test/Microsoft.AspNetCore.Mvc.FunctionalTests/MvcTestFixture.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
```

Comment:
I don't think so, it's most likely left over, we only add TagHelperFeature when you invoke AddTagHelpersAsServices()